stories/panel/panel.stories.js

```diff
@@ -197,3 +197,64 @@ fixedHeightContent.parameters = {
         iframeHeight: 300
     }
 };
+
+export const rtl = () => `
```

Comment:
<img width="1505" alt="Screen Shot 2020-11-06 at 11 14 23 AM" src="https://user-images.githubusercontent.com/39598672/98388857-45ae6a00-2021-11eb-825b-43cbf0939a4c.png">

You don't have to add this example, we have a "switcher" on the top of the doc page. All examples should apply the RTL rules when you switch to this mode.